app/controllers/emails_controller.rb

```diff
@@ -1,40 +1,25 @@
-class PagesController < ApplicationController
+class EmailsController < ApplicationController
   before_action :authenticate_user!
+  before_action :initialize_filters
 
   def inbox
-    @filters = ["Inbox", "Reply later", "All"]
-
     # TODO: This is a temporary solution to retrieve the last 20 emails
     # TODO: Add a user setting to toggle email servers
     @email_server = current_user.email_servers.first
     @email_threads = @email_server.email_threads.includes(:messages).last(20)
-    # @emails = @email_server.retrieve_recent_emails if @email_server.present?
-  end
-
-  def done
-  end
-
-  def team
-  end
-
-  def calendar
-  end
-
-  def vip
   end
 
-  def sent
+  def reply_later
+    # TODO: To be implemented later
   end
 
-  def later
+  def all
+    # TODO: To be implemented later
   end
 
-  def draft
-  end
+  private
 
-  def archive
-  end
-
-  def trash
+  def initialize_filters
```

Comment:
This is great. Eventually we won't need these filters, but it works for now. We will want to initialize custom filters in the future, though, so I like this route.